Wrappers/Python/test/test_cropped_readers.py

```diff
@@ -0,0 +1,79 @@
+import os
+import unittest
+
+import numpy as np
+import vtk
+from ccpi.viewer.utils.conversion import (Converter, cilRawCroppedReader,
+                                          cilMetaImageCroppedReader,
+                                          cilNumpyCroppedReader)
+
+
+
+class TestCroppedReaders(unittest.TestCase):
```

Comment:
I would add a test for making sure the read data are correct too.

Reply:
Added a check for this